unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/GlobalScene.cs

```diff
@@ -15,7 +16,7 @@ public class GlobalScene : ParcelScene
 
         public override bool IsInsideSceneBoundaries(Vector2Int gridPosition, float height = 0) { return true; }
 
-        public override void SetData(LoadParcelScenesMessage.UnityParcelScene data)
+        public override async UniTask SetData(LoadParcelScenesMessage.UnityParcelScene data)
```

Comment:
Global scenes also use asset bundles! The avatars scene for example, needs the AB url for the wearables @Kinerius